src/QsCompiler/Tests.LanguageServer/Tests.cs

```diff
@@ -233,8 +238,8 @@ async Task RunTest(bool emptyLastLine)
                     Assert.AreEqual(expected, got);
                 }
             }
-            await RunTest(emptyLastLine: true);
-            await RunTest(emptyLastLine: false);
+            await RunTest(emptyLastLine: true, useQsExtension: false);
```

Comment:
It would be good to also run these tests when the files have a .qs extension.

Reply:
Yes, I agree. As I mentioned in the PR description, that works fine for me when running the tests locally but fails in the CI build for some reason that I haven't figured out yet. (If you look at the next-to-last commit of this PR you'll see what I tried.)

I will investigate and get that added ASAP, but since this unblocks the tests I was thinking to get this merged first. Let me know if you agree.